sdk/core/azure-core/src/main/java/com/azure/core/implementation/RestProxy.java

```diff
@@ -141,7 +141,6 @@ public Object invoke(Object proxy, final Method method, Object[] args) {
                 request = createHttpRequest(methodParser, args);
                 Context context = methodParser.context(args).addData("caller-method", methodParser.fullyQualifiedMethodName());
                 context = startTracingSpan(method, context);
-
```

Comment:
Any need for this change?

Reply:
Will clean up the code for testing. Thanks for catching this.